### PR TITLE
fix(ga4 and ua): cookie conflicts

### PR DESCRIPTION
**Pull request by mihir-4116**
## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by MoumitaM — APPROVED** [review]

File: src/integrations/GA4/browser.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default class GA4 {
     if (this.analytics.userId) {
       gtagParameterObject.user_id = this.analytics.userId;
     }
-
+    gtagParameterObject.cookie_prefix = 'rs';
```
Comment (krishna2020) [inline src/integrations/GA4/browser.js:50]: Any test cases to add/update?

Comment (ItsSudip) [inline src/integrations/GA4/browser.js:50]: Have we verified this will stop the conflict between ga4 and ga ua?

Comment (mihir-4116) [reply src/integrations/GA4/browser.js:50]: yes @ItsSudip

**Review by ItsSudip — APPROVED** [review]

**Review by krishna2020 — APPROVED** [review]

File: src/integrations/GA4/browser.js (hunk 1 above)
Comment (krishna2020) [reply src/integrations/GA4/browser.js:50]: Test cases will be added in next PR

**Pull request merged**